openbook_auth/models.py

```diff
@@ -1996,8 +2069,22 @@ def _send_post_comment_push_notification(self, post_comment, notification_messag
                                                                  target_user=notification_target_user)
 
     def _delete_post_comment_notification(self, post_comment):
-        PostCommentNotification = get_post_comment_notification_model()
-        PostCommentNotification.delete_post_comment_notification(post_comment_id=post_comment.pk,
+        if post_comment.parent_comment is not None:
+            PostCommentNotification = get_post_comment_notification_model()
+            PostCommentNotification.delete_post_comment_notification(post_comment_id=post_comment.pk,
```

Comment:
`delete_post_comment_notification(post_comment_id=post_comment.pk,` this was already there. I now just check if a `parent_comment` exists, then its a reply so I delete the reply notification otherwise the usual comment notification.

Reply:
Aha, missed the difference of `PostCommentReplyNotification` vs `PostCommentNotification`